src/handlers/handlers.rs

```diff
@@ -86,3 +101,29 @@ pub async fn index_page() -> Result<HttpResponse, Error> {
         .content_type("text/html; charset=utf-8")
         .body(html_str))
 }
+
+#[get("/v1/stats/all")]
+async fn status_get_api() -> impl Responder {
+    let sys = System::new();
+
+    match sys.cpu_temp() {
+        Ok(cpu_temp) => println!("\nCPU temp: {}", cpu_temp),
+        Err(x) => println!("\nCPU temp: {}", x),
+    }
```

Comment:
We can maybe remove this block, as it just acts like a switch case, eventually printing values to the console.